sky/packages/sky/lib/src/widgets/snack_bar.dart

```diff
@@ -71,15 +85,15 @@ class SnackBar extends Component {
     ];
     if (actions != null)
       children.addAll(actions);
-
     return new SlideTransition(
```

Comment:
SlideInTransition wouldn't change - still a Widget. AnimatedComponent (or whatever) would just be a convenience wrapper that manages the performance for you.